pkgs/development/compilers/dotnet/build-dotnet.nix

```diff
@@ -77,24 +73,30 @@ stdenv.mkDerivation rec {
     runHook postInstall
   '';
 
-  postFixup = lib.optionalString stdenv.isLinux ''
-    patchelf --set-interpreter "${stdenv.cc.bintools.dynamicLinker}" $out/dotnet
-    patchelf --set-rpath "${rpath}" $out/dotnet
-    find $out -type f -name "*.so" -exec patchelf --set-rpath '$ORIGIN:${rpath}' {} \;
-    find $out -type f \( -name "apphost" -or -name "createdump" \) -exec patchelf --set-interpreter "${stdenv.cc.bintools.dynamicLinker}" --set-rpath '$ORIGIN:${rpath}' {} \;
-
-    wrapProgram $out/bin/dotnet \
-      --prefix LD_LIBRARY_PATH : ${icu}/lib
-  '';
-
   doInstallCheck = true;
   installCheckPhase = ''
-    # Fixes cross
-    export DOTNET_SYSTEM_GLOBALIZATION_INVARIANT=1
```

Comment:
Have you tested cross without this?

Reply:
No.  Can you point me at example of how to test this?  I'm not really clear on what cross configs are supported.

Reply:
Try `nix build -Lf . pkgsCross.aarch64-multiplatform.dotnet-sdk`, that what i tested against before.

Reply:
This works, but `doInstallCheck` seems to be forced off in cross, so it's not actually running anything.  I think that makes sense, since it would be an aarch64 elf.

Reply:
Also worth mentioning that the reason I wanted to remove it is that it stops dotnet from loading icu, which is a useful thing to test.

Reply:
> This works, but doInstallCheck seems to be forced off in cross, so it's not actually running anything. I think that makes sense, since it would be an aarch64 elf.

Ah yeah, I guess that only happened for me because of binfmt.